controlpanel/kubeapi/views.py

```diff
@@ -1,30 +1,16 @@
 from functools import wraps
-import os
 from urllib.parse import urljoin
 
 from django.conf import settings
 from django.core import exceptions
 from django.views.decorators.csrf import csrf_exempt
 from djproxy.views import HttpProxy
-import kubernetes
 
+from controlpanel import api
```

Comment:
Would this not be better as `from controlpanel.api import kubernetes`?

Reply:
As discussed, I've imported it this way to avoid ambiguity so that it's not confused with the official `kubernetes` module.